chingu-voyages/v52-tier3-team-38
# Added address field to list view, adjusted page.js to show list view …

# Description
This PR fixes bugs that occurred/ were present during today's meeting.

## Related to the following Jira/Trello issue 
N/A

## Main changes explained:
- When an admin is logged in they initally see the list view instead of the default template for the admin page.
- Each card in the list view now shows all appointment information, including the missing address.
- Removed AdminPage as it was no longer needed.

## How to test:
1. check into current branch
2. do `npm install` and `npm run dev` to run this PR locally
3. Login as an admin, the first page you see should be the list view, check to make sure the address field is present in each card.

## Note:
I believe the client error that occured when signing up happened due to me deleting an existing appointment from the database while the form was active. I have not been able to duplicate that error as the functionality now works as intended both locally and in production.
